packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -327,19 +328,20 @@ export function revisionMetadataSourceFromInfo(
 	return { getIndex, tryGetInfo, hasRollback };
 }
 
+// TODO: Should this return a tagged change?
```

Comment:
Looks like either way is fine. I think doing so make the API a little nicer, and seem less wastefull (the code in treeCheckout should be updated to call mintRevisionTag much earlier, so it doesn't have to create anonymous changes)